lms/templates/view.html

```diff
@@ -28,23 +29,39 @@ <h1>{{ _('תצוגת תרגיל') }} {{ solution['exercise']['id'] }}: {{ soluti
       {% endfor -%}
       </ul>
     </nav>
-    {% if is_manager %}
+    {% if config.SHAREABLE_SOLUTIONS %}
```

Comment:
Add: if this is the user/the admin.

We shouldn't allow other users to delete the share link.

Reply:
The won't see it because of the condition in line 20